app/src/main/java/edu/uw/tcss450/chatapp_group1/MainActivity.java

```diff
@@ -277,7 +290,24 @@ private void stopLocationUpdates() {
         mFusedLocationClient.removeLocationUpdates(mLocationCallback);
     }
 
-
+    /**
+     * method thats responsible for changing Latitude and longitude to zipcode
+     * @return currentZipCode
+     */
+    public String getCurrentZip(){
+        currentLatitude = mLocationModel.getCurrentLocation().getLatitude();
+        currentLongitude = mLocationModel.getCurrentLocation().getLongitude();
+        Geocoder geocoder = new Geocoder(getBaseContext(), Locale.ENGLISH);
+        try {
+            List<Address> addresses = geocoder.getFromLocation(currentLatitude, currentLongitude, 1);
+            Address returnAddress = addresses.get(0);
+            currentZipcode = returnAddress.getPostalCode();
+            Log.d("LOCATION FROM MainActivity/getCurrentZip", currentZipcode);
```

Comment:
you should get rid of this, if this is for testing purposes

Reply:
ok. I will.